examples/node/cli.rs

```diff
@@ -54,13 +54,27 @@ fn node_info(node: &LightningNode) {
         "Node PubKey: {}",
         PublicKey::from_slice(&node_info.node_pubkey).unwrap()
     );
-    println!("Number of channels: {}", node_info.num_channels);
+    println!("Number of connected peers: {}", node_info.num_peers);
+    println!(
+        "       Number of channels: {}",
+        node_info.channels_info.num_channels
+    );
```

Comment:
I just had some fun playing around with this: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=ccc0e725fcfe66ba92cc259971f0cb47

Reply:
Nope.